bot/helper/ext_utils/torrent_search_utils.py

```diff
@@ -0,0 +1,19 @@
+import requests
+from bot import LOGGER
+
+def search_torrent_link(query):
+	LOGGER.info("Searching torrent on query")
+	query = query.replace("/", "")
+	try:
+		r = requests.get(
+        	"https://torrent-paradise.ml/api/search?q=" + query)
+		torrents = r.json()
+		torrents = sorted(torrents, key=lambda i: i['s'], reverse=True)
+		torrent = torrents[0]
```

Comment:
This assumes only one torrent is returned by the api for the query but it can return multiple. Won't it make more sense if it shows the name of the torrents to the user and let the user decide what he/she wants to mirror instead of just assuming it to be the first one from the sorted list